pkg/kapp/clusterapply/cluster_change_set.go

```diff
@@ -98,15 +100,22 @@ func (c ClusterChangeSet) Apply(changesGraph *ctldgraph.ChangeGraph) error {
 		expectedNumChanges, c.opts.ApplyingChangesOpts, c.clusterChangeFactory, c.ui)
 	waitingChanges := NewWaitingChanges(expectedNumChanges, c.opts.WaitingChangesOpts, c.ui)
 
+	var unsuccessfulChanges []string
+
 	for {
-		appliedChanges, err := applyingChanges.Apply(blockedChanges.Unblocked())
+		appliedChanges, unsuccessfulChangeDesc, err := applyingChanges.Apply(blockedChanges.Unblocked())
 		if err != nil {
 			return err
 		}
```

Comment:
Should the timeout errors also include the unsuccessfulChanges?

Reply:
I think they should!

Reply:
I am still thinking if should do this and what would be the best way to present the errors.